plugins/view-resources/src/components/list/ListHeader.svelte

```diff
@@ -61,8 +61,9 @@
 
   let accentColor: ColorDefinition | undefined = undefined
 
+  $: showColors = (viewOptions as any).shouldShowColors
```

Comment:
```suggestion
  $: showColors = (viewOptions as any).shouldShowColors !== false
```
viewOptions?.shouldShowColors could be undefined, in this case it need to be true.